### PR TITLE
136 improve post UI + 135 discussion side bar

### PR DESCRIPTION
**Pull request by Tguntenaar**
[PREVIEW](https://near.org/thomasguntenaar.near/widget/gigs-board.pages.community.activity?handle=zero-knowledge)

_Resolves #135_
_Resolves #136_
Solves issue #135 and #136 

I just topped up my Near wallet to be able to publish the widgets but it the bank transfer is taken some time, so we will have to work with this screenshot for now:

Screenshot 2023-07-16 at 14 55 56

### CODE REVIEW

File: src/gigs-board/entity/post/Post.jsx [hunk 1]
```diff
@@ -53,12 +53,16 @@ function href(widgetName, linkProps) {
 /* END_INCLUDE: "common.jsx" */
 /* INCLUDE: "core/lib/gui/attractable" */
 const AttractableDiv = styled.div`
-  box-shadow: 0 0.125rem 0.25rem rgba(0, 0, 0, 0.075) !important;
+  // box-shadow: 0 0.125rem 0.25rem rgba(0, 0, 0, 0.075) !important;
```
Comment (ailisp) [inline src/gigs-board/entity/post/Post.jsx:56]: let's remove the commented code

File: src/gigs-board/pages/community/activity.jsx [hunk 2]
```diff
@@ -101,7 +101,7 @@ const CommunityActivityPage = ({ handle }) => {
   }
 
   const communityData = DevHub.get_community({ handle });
-
+  console.log({ communityData });
```
Comment (ailisp) [inline src/gigs-board/pages/community/activity.jsx:104]: Remove this?

**Review by ailisp — APPROVED** [review]
Looks good! And the screenshot looks very nice to me.  @ori-near can you confirm the post UI and side bar looks good to you? (once Thomas deploy the preview)

**Comment by ori-near** [comment]
Hi @ailisp and @Tguntenaar – can you please make the top elements after the navigation bar more vertically aligned? The green represents aligned, the red represents misaligned: 

Screenshot 2023-07-19 at 11 48 34 AM

**Comment by Tguntenaar** [comment]
@ori-near Alright will fix this!

**Comment by Tguntenaar** [comment]
@ori-near the result

Screenshot 2023-07-24 at 14 57 55

**Comment by Tguntenaar** [comment]
@ori-near @ailisp I finally deployed the [preview](https://near.org/thomasguntenaar.near/widget/gigs-board.pages.community.activity?handle=zero-knowledge). Sorry for the wait

**Comment by ailisp** [comment]
Excellent!

**Comment by Tguntenaar** [comment]
@ori-near @ailisp Is this ready to be merged?

**Comment by ailisp** [comment]
@ori-near Can you take a look? The "Zero Knowledge" and post button is not aligned on top, but the card top of "Zero Knowledge" is aligned with the button top. Previously there is no card boarder, this seems fine to me.

**Comment by Tguntenaar** [comment]
@ori-near could you review this PR?

**Comment by ori-near** [comment]
Hi @Tguntenaar – thanks so much! Overall looks good. A few feedback items:
- For the hover drop shadow (for posts and sidebar items), can we do something a little less visually distracting? Maybe 
`rgba(48, 48, 48, 0.15) 0px 0.1rem 0.1rem !important`
- Right now I don't see any of the hover effects for the interaction buttons (like, reply, etc). Can we bring them back?
- Are we still going to show the show the Read More expansion box?
- Can you introduce a little bit of spacing bellow the title of the post and the content, similar to the design in https://github.com/near/neardevhub-widgets/issues/135. Can you also slightly reduce the spacing between each posts?

**Comment by Tguntenaar** [comment]
> Are we still going to show the show the Read More expansion box?

What do you mean by this? @ori-near 

> Right now I don't see any of the hover effects for the interaction buttons (like, reply, etc). Can we bring them back?

Which color would you like the hover effect?

Right now I just used black which is a bit harsh I would say:

Screenshot 2023-08-02 at 20 24 55

**Comment by Tguntenaar** [comment]
@ori-near Ah I see the designs used a different approach 'see more' on the far left side in bolt. I implemented that instead. Do you want me to switch back to the read more approach? The rest of the points have been fixed.

**Comment by ori-near** [comment]
Hi @Tguntenaar – I'm fine with the design approach, I just didn't see any of the "load more" rendering in your preview, so wanted to make sure the functionality was still in place. Thank you.

**Comment by Tguntenaar** [comment]
@ori-near I made a solution post with enough text that the 'load more' option is visible and testable. I will make sure to include 'edge' cases in my previews from now onl.

**Comment by ori-near** [comment]
Hi @Tguntenaar – no worries, thanks so much! Can we make the see more happen a little before? So that we show about one paragraph worth of content ... 
Screenshot 2023-08-02 at 2 04 51 PM

Also I'm still not seeing the hover effect on the interaction buttons. Thanks again!

**Comment by Tguntenaar** [comment]
@ori-near Alright! I didn't change any of that logic so it was the same as before. The implementation displays the content up to the 5th newline in the description text. This behavior likely occurs because most posts begin with additional new lines due to introductions or other formatting, rather than having continuous Lorem Ipsum paragraphs.

Anyway, I just made an adjustment to cut of the text after the 3rd newline/ enter.

Also I updated the [Preview](https://near.org/thomasguntenaar.near/widget/gigs-board.pages.community.activity?handle=zero-knowledge) version to reflect all changes.

**Comment by ori-near** [comment]
Hi @Tguntenaar looks good, thank you!

**Comment by ailisp** [comment]
I double checked with the acceptance criteria of #135 and #136. It looks good to me! Thanks @Tguntenaar to bring this to real!

**Pull request merged**